### PR TITLE
[DevOps] Update LLM packer scripts

### PR DESCRIPTION
**Pull request by DinukaH2O**
### Description

- Added a string parameter to checkout to the specified branch tag.
- Changed port to 80 using port forwarding(nginx)
- Added GPU support 
- Built and tested cloud images

### CODE REVIEW

File: jenkins/packer/llm-studio-cloud-images.groovy [hunk 1]
```diff
@@ -5,6 +5,7 @@ properties(
         parameters(
             [
                 string(name: 'BRANCH_TAG', defaultValue: 'origin/main'),
+                string(defaultValue: 'v0.1.0', description: 'eg: v0.1.0', name: 'BRANCH_VERSION', trim: true),
```
Comment (pascal-pfeiffer) [inline jenkins/packer/llm-studio-cloud-images.groovy:8]: hardcoded intended?

File: jenkins/packer/install_llm_studio_ubuntu2004.sh [hunk 2]
```diff
@@ -17,9 +18,33 @@ sudo apt -y install python3.10
...
+
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
+sudo mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
+wget https://developer.download.nvidia.com/compute/cuda/11.8.0/local_installers/cuda-repo-ubuntu2004-11-8-local_11.8.0-520.61.05-1_amd64.deb
+sudo dpkg -i cuda-repo-ubuntu2004-11-8-local_11.8.0-520.61.05-1_amd64.deb
+sudo cp /var/cuda-repo-ubuntu2004-11-8-local/cuda-*-keyring.gpg /usr/share/keyrings/
+sudo apt-get update
+sudo apt-get -y install cuda
+
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID) && curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey \
```
Comment (mmalohlava) [inline jenkins/packer/install_llm_studio_ubuntu2004.sh:34]: Minor: quotes are missing around "$ID$VERSION" - just safer.

File: jenkins/packer/install_llm_studio_ubuntu2004.sh [hunk 3]
```diff
@@ -17,9 +18,33 @@ sudo apt -y install python3.10
...
+
+sudo apt-get -y update
+sudo apt-get install -y nvidia-container-runtime
+rm cuda-repo-ubuntu2004-*.deb
+
+
 # Clone h2o-llmstudio
 git clone https://github.com/h2oai/h2o-llmstudio.git
 cd h2o-llmstudio
+git checkout "$VERSION"
```
Comment (mmalohlava) [inline jenkins/packer/install_llm_studio_ubuntu2004.sh:46]: Nitpick: you can make it faster by cloning only specific tag/branch:
See: https://stackoverflow.com/questions/20280726/how-to-clone-a-specific-git-tag

**Review by mmalohlava — APPROVED** [review]
minor comments - pls incorporate them and merge.

**Review by hasithjp — APPROVED** [review]

**Pull request merged**